subprojects/core/src/main/java/org/gradle/api/tasks/AbstractExecTask.java

```diff
@@ -133,7 +133,7 @@ public T setArgs(@Nullable Iterable<?> arguments) {
     @Optional
     @Input
     @Override
-    @ToBeReplacedByLazyProperty
+    @ToBeReplacedByLazyProperty(skipForReport = true, comment = "Skipped for reporting since setter is using generics")
```

Comment:
❓ Why are we skipping this?

Reply:
I guess these new methods are benign enough, though they _can_ cause a plugin compiled against them not to work with an earlier Gradle version. Again, that's not something we say we'd support, but just to be aware...

I think it's acceptable and this will not burn us in the future. And I'm hoping I will not need to revisit this comment later. :)